webdriverio/webdriver/examples/deque/test/specs/login.accessibility.spec.ts

```diff
@@ -0,0 +1,9 @@
+describe('Accessibility Spec', () => {
+
```

Comment:
nitpicking, but maybe remove this line break

Reply:
removed line break